java/code/src/com/suse/manager/webui/controllers/admin/mappers/ResponseMappers.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2021 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package com.suse.manager.webui.controllers.admin.mappers;
+
+import com.redhat.rhn.common.localization.LocalizationService;
+import com.redhat.rhn.domain.cloudpayg.PaygSshData;
+
+import com.suse.manager.webui.controllers.admin.beans.PaygFullResponse;
+import com.suse.manager.webui.controllers.admin.beans.PaygProperties;
+import com.suse.manager.webui.controllers.admin.beans.PaygResumeResponse;
+import com.suse.manager.webui.utils.ViewHelper;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ResponseMappers {
```

Comment:
Should this class be renamed/moved to clarify it is specific to PAYG pages? I find it a bit misleading that the title is very generic, and placed in a very generic package.

Of course a different thing would be if more controllers delegated some processing to mappers - but IIUC this is really the first such case, right?

Reply:
There are other cases that also delegate the mapping to an external class [1] [2].
I agree with you that the name is too generic. I will remane it to `PaygResponseMappers.java`.

[1] https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/controllers/clusters/mappers/ResponseMappers.java
[2] https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/controllers/contentmanagement/mappers/ResponseMappers.java